templates/base.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+  <head>
+    {% block head %}
+  <link rel="stylesheet" href="static/styles.css">
+  <title>{% block title %} Seats held by women {% endblock %}</title>
+    {% endblock %}
+  </head>
```

Comment:
I think it should be something like

```
  <head>
    {% block head %}
    <link rel="stylesheet" href="static/styles.css" />
    <title>{% block title %} Seats held by women {% endblock %}</title>
    {% endblock %}
  </head>
```

Reply:
BTW, does the `{% block %}` part is necessary here? It's not like the values are changing here on page reload.